catalog/MDCCatalog/MDCCatalogTileView.swift

```diff
@@ -63,10 +63,16 @@ class MDCCatalogTileView: UIView {
     guard !bounds.isEmpty else {
       return
     }
+    //imageView.tintColor = AppTheme.globalTheme.colorScheme.primaryColor
```

Comment:
is this supposed to be commented out or deleted?

Reply:
Removed.